src/npc.cpp

```diff
@@ -1112,7 +1112,7 @@ bool npc::wear_if_wanted( const item &it, std::string &reason )
                 return armor.covers( bp );
             } );
             if( iter != worn.end() && !( is_limb_broken( bp ) && iter->has_flag( flag_SPLINT ) ) ) {
-                took_off = takeoff( *iter );
+                took_off = takeoff( *iter ); //Can I turn this *iter into an item_location?
```

Comment:
Since `iter` is always from `worn`, I think it's fine to just create the item location as `item_location( *this, &*iter )`.

Reply:
Thank you for looking at this. I didn't know you could use both & and * together like that and knowing that helps me not do extra unnecessary steps. Now that I know how easy it is to make an `item_location` object I am rethinking my approach here.